Qt.py

```diff
@@ -839,6 +837,45 @@ def _log(text):
         sys.stdout.write(text + "\n")
 
 
+def __loadUi(fname, baseinstance=None):
+    """
+    Dynamically load a user interface from the given ``uifile``.
```

Comment:
We'll need to move this documentation into the [README](https://github.com/mottosso/Qt.py#documentation) somehow, maybe think about a different layout since there is a quite a bit of it.

Alternatively, link to the PySide2 documentation for it as they should match, and highlight that the `custom` argument is excluded.